---
title: Missense mutations in genes Chd2 and *Myocd* interacting with *Myod*1 likely cause abnormal
  skeletal phenotypes that predispose preterm birth by premature rupture of membranes.
author: "Ruth Raichur"
date: '2021-04-30'
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

## Summary

The purpose of this study was to postulate an underlying cause of a patient’s phenotype that is likely genetic. Missense mutations were identified in a 13 year old XY individual (Decipher ID: <a href="https://www.deciphergenomics.org/patient/290490/overview/general"> 290490 </a>) in two genes presenting multiple abnormal skeletal phenotypes such as protruding ears, long face, long philtrum, short nose, low posterior hair line and long fingers as well as preterm birth by premature rupture of membranes. Thus, an interaction between the two mutated genes was hypothesized to cause phenocopies of connective tissue disorders known to predispose premature birth by premature rupture of membranes. This was done by evaluating the severity of the missense mutations in both genes using Decipher and UCSC genome browser. Then, further research was evaluated to understand the relationship and functions of the affected genes in this specific patient. Missense mutations in *Chd2* and *Myocd* genes have been found as a likely cause for multiple abnormal skeletal developmental phenotypes such as the ones presented in this patient. The gene interactions with *Myod*1, a skeletal muscle regulatory factor, appears to link pathways of these mutated genes suggesting a causative effect on the resultant phenotype seen in the patient. The inheritance of these mutations in these genes is identified to be most likely de-novo as they are both rare and missense SNPs. A potential explanation for the phenotypes mimicking connective tissue disorders lies in the roles these two genes play in skeletal and muscle differentiation since it is known to be closely linked to connective tissue differentiation.

## Introduction

The patient of study presents with multiple abnormal developmental skeletal phenotypes such as protruding ears, long face, long philtrum, short nose, low posterior hair line and long fingers. The impacts of these phenotypes on the quality of life are minimal as these are not seen to affect internal functions. The most intriguing phenotype is premature birth following premature rupture of fetal membranes. Missense variants in genes *Chd2* and *Myocd* were found, both of which likely explain most of the physical abnormal phenotypes and will be explained in this paper. 

Premature rupture of membranes (PROM) is defined by the rupture of the fetal membranes prior to the onset of labor (Tchirikov et.al, 2017). If the rupture occurs prior to 37 weeks, it is classified as preterm premature rupture of membranes (pPROM) (Khade and Bava, 2018). A study in 2018 published by the International Journal of Reproduction, Contraception, Obstetrics and Gynecology found that pPROM increases the risk of fetal prematurity, preterm birth (PTB) as well as lead to perinatal morbidity and neonatal complications. pPROM also posed a 1-2% risk of fetal death (Khade and Bava, 2018). While pPROM and PTB are recognized as separate entities, pPROM often leads to PTB suggesting a strong link in causal factors (Capece et.al, 2014). Research found that preterm birth (PTB) is the first cause of infant mortality and morbidity (Di Renzo et.al, 2018). The risk factors known to be associated with pPROM and PTB include socioeconomic, environmental and genetic factors of both mother and fetus. For the purpose of this study, fetal conditions such as preeclampsia and fetal malformations (Di Renzo et.al, 2018) as well as genetic factors will be the main focus of consideration. This however does not rule out the interactions among risk factors when understanding the etiology of pPROM. Because of this, it is difficult to determine the true genetic basis of the preterm birth by premature rupture of membrane. 

Previous genome wide association studies (GWAS) have determined PTB to be a heterogenous phenotype. In addition, both maternal and fetal genomes are shown to increase susceptibility (Bezold et.al, 2013). Based on a parametric linkage analysis study followed by a case-control study and haplotype segregation analysis, a novel susceptibility gene (IGF1R) was identified through the fetal genome, that was seen to predispose the fetus to preterm birth (Alleman et.al, 2012). Focusing on pPROM, while most single nucleotide polymorphisms (SNPs) known to cause this disease are maternal. Fetal genetic factors such as Ehlers-Danlos Syndrome and Marfan syndrome, a connective tissue disorder associated with physical abnormalities, are known to predispose pPROM suggesting a principle genetic role (Romero et.al, 2010). On a genomic level, genetic variations like SNPs in both the fetal and maternal DNA plays a significant role in predisposition to both pPROM and PTB.

While at first glance the mutations in *Chd2* and *Myocd* do not seem to explain premature birth following premature rupture of fetal as these genes are more strongly associated with skeletal muscle development. Analysis of pathways, interactions between these genes and the effects of the mutations may be useful in explaining this phenotype. The possibility of PTB being a resultant phenotype of the abnormal developmental phenotypes is also one that must be taken into consideration. 

## Description of Findings

Premature birth following premature rupture of fetal membranes is a phenotype that is unique to the patient given the known missense variants in *Chd2* and *Myocd* genes. Neither mutations nor genes have been identified as PTB or pPROM causing genes. There is perhaps an interesting link between the other phenotypes of abnormal physical development of the face, ears, philtrum (vertical groove between the base of the nose and the border of the upper lip), nose, and fingers that could possibly explain PTB and pPROM since many studies have seen a predisposition to these pPROM due to fetal genetic developmental factors (Romero et.al, 2010). 

This patient presents phenotypes that are commonly related to fetal genetic disorders (that are known to predispose pPROM) such as Ehlers-Danlos Syndrome (EDS) and Marfan syndrome (MFS). They are both connective tissue disorders that have many overlapping phenotypes. MFS and EDS phenotypes, while highly variable, include skeletal manifestations like long bone overgrowth as well as craniofacial features which are physical abnormalities seen in the patient of study (Meester et.al, 2017). Interestingly, neither *Chd2* nor *Myocd* have been previously found to be disorder causing genes in studies nor were they identified to be directly related to pPROM or PTB in humans (Romero et.al, 2010, Capece et.al, 2014). However, profound defects in the yolk sac vasculature and evidence of generalized intrauterine growth retardation were seen in CMV-Cre+/*Myocd*F/Fmutant mice embryos which may be grounds for identifying the mutation in *Myocd* as the underlying genetic cause of this phenotype seen in the patient (Huang et.al, 2008).

The patient is seen to have missense mutations in *Chd2* and *Myocd* with variable effects as seen on Decipher (Figures 1 and 2). The *Chd2* gene encodes chromodomain helicase DNA-binding protein 2 (OMIM 602119). The G to A mutation on *Chd2* is observed to be less intolerant according to the missense constraint ranges. This range describes the tolerance of this gene to the missense variant in preserving thier function. Further, the codon in which this mutation is located codes for amino acid Arginine that is highly conserved across species (Figure 1). This mutation is seen to change the charged Arginine amnio acid to an uncharged Glutamine. *Myocd* encodes for Myocardin, a transcriptional coactivator of serum response factor (SRF) (OMIM 606127). The C to T mutation of *Myocd* is observed to be more intolerant according to the missense constraint ranges. This C to T mutation causes a change from a non-polar Proline that is highly conserved within species to a polar Serine (Figure 2). We see high conservation of the amino acid sequences across multiple species which is an important note when comparing effects on gene mutants in non-human studies. Neither of these mutations in these gene have matching Copy Number Variants (CNV) syndromes on Decipher, making CNV data irrelevant to this study.

In terms of the effect these mutations have on the protein, we see on Decipher that identified mutations on MYCOD are commonly protein changing while those on *Chd2* is 52% is likely loss of function while the 48% is protein changing. However, when filtered out by phenotype, *Chd2* is only observed to causes protein changes. This concurs with what is seen in the patient given that both mutations are missense in exon regions, suggesting a potential link between the effect on change of protein to similar phenotype expressed. Further analysis of the structures of these proteins would provide conclusive findings on the true effects of these missense mutations as these substitutions involve switches between amino acids with significantly different properties that could potentially elicit these phenotypic responses. 

Analysis on UCSC Genome Browser SNP data shows that both mutations likely occur spontaneously. gnomAD data describes the occurrence of sequence variants of these two genes. This data is visualized on genome browser for each gene and these missense mutations are seen to be unique SNPs. *Chd2* is seen to have another missense mutation at the same position chr15: 93000537 (Figure 3). Unfortunately, this patient data was not available on Decipher for comparison. The SNP data for *Myocd* shows no other mutation in the same position as the patient of the study. There is a higher frequency of silent mutations present on the gene compared to missense mutations which concurs with the missense tolerance scale on Decipher which revealed that *Myocd* was highly intolerant (Figure 4). gnomAD data reveal both mutations to be rare alleles as the G to A mutation in the observed position of *Chd2* is seen in 1 in 152096 alleles and the C to T in *Myocd* seen in 1 in 152212 (Figure 5). This is strong evidence to conclude that both these mutations are rare and are likely de-novo leaving much to be studied. 

## Discussion and Conclusion 
	
The deeper study into the functions of these genes and resultant phenotypes in these mutations should provide a better understanding of a potential link we see between the abnormal physical phenotypes and the premature birth due to premature rupture of membranes. 

According to OMIM, *Chd2* encodes for the chromodomain helicase DNA-binding protein 2 and is involved in myogenesis via the interaction with *Myod1*, a skeletal muscle-specific myogenic regulatory factors (OMIM 602119). *Chd2* was found to interact with *Myod1* in gene regulatory sequencing by binding to myogenic promoters. This close relationship between the interaction of *Myod*1 and *Chd2* suggests *Chd2*’s role in skeletal muscle differentiation (Harada et.al, 2012). Molecular genetic analyses of mutations in this gene in humans have primarily described it as a de-novo mutation with numerous postulations of haploinsufficiency as the disease mechanism while its most common disease phenotype is related to epilepsies and learning disabilities. Further, the region *Chd2* is located in, is implicated in a rare genetic disorder that presents phenotypes of growth retardation, cardiac defects, and early postnatal lethality (Carvill et al., 2013). In mice, a single knockout of *Chd2* resulted in pronounced lordokyphosis, reduced body fat, postnatal runting, and growth retardation suggesting *Chd2* haploinsufficiency resulting in abnormal human phenotypes as seen in this patient (Kulkarni et.al, 2008). 

*Myocd* encodes Myocardin, a smooth and cardiac muscle-specific transcriptional coactivator of serum response factor (SRF) (OMIM 606127). Common phenotypes associated with mutations in this gene was congenital megabladder formation. Studies have found genetic interactions between myocardin and myogenic basic helix-loop-helix transcription factors *Myod* (or *Myod1*), the gene that is also seen to interact with *Chd2*. *Myocd* was found to repress the skeletal muscle differentiation program, overwriting the function of *Myod1* which suggests that transcriptional coregulator *Myocd* has the ability to act as a bifunctional molecular switch in regulating the smooth and skeletal muscle phenotypes (Long et.al, 2007). Perhaps the missense mutation observed in the patient has resulted in the overwriting of *Myod*1 and a mis-regulation of skeletal muscle development by *Myocd*.

A possible mechanism by which the mutation affects *Myocd* is by affected the isoform interactions of myocardin this gene codes for. A mice model studied the effects of the four isoforms on the MEF2C-mediated *Myod1* suppression. All four isoforms of myocardin played a significant role in this suppression, overwriting the skeletal muscle program of differentiation (Imamura et. al, 2010). Given that all four isoforms play a significant role, it is possible that the mutation in this gene seen in the patient affects the structure of an isoform. This change could affect its interaction with the isoforms and their role in *Myod1* suppression causing abnormalities in musculoskeletal development.

The patient presents with multiple phenotypes that are likely a result in the mis regulation in musculoskeletal system and skeletal development as seen by the abnormalities in development in the ears, face, philtrum, nose and fingers. While alone, these two genes have minimal associations with skeletal development, there is strong evidence to support the hypothesis that these missense mutations in *Chd2* and *Myocd* could likely cause these phenotypes linked by their common involvement with *Myod1* that is a known a skeletal muscle-specific myogenic regulatory factors (OMIM 602119). A possible explanation of for the PTB by pPROM phenotype in the patient could be a resultant phenotype of the other abnormal skeletal developmental phenotypes this patient presents with. This is a fair conclusion since, studies have shown that abnormal skeletal development have shown the predispose preterm birth by pPROM (Meester et.al, 2017). 

A further point of inquiry would be establishing the true mechanism by which this abnormal skeletal development and pPROM occurs. By analyzing the cell differentiating roles of MYCOD and *Chd2*, we could determine a connection between the interactions of these genes and resultant phenocopies of connective tissue disorders such as EDS and MFS that are known to predispose pPROM which is seen in this patient. There are considerable grounds to draw this conclusion since the proper development and survival of connective tissues is dependent on muscle and skeletal tissue development (Li et.al, 2010). Further there is considerable overlap in expression of these genes in differentiating cardiac, skeletal and muscle tissue especially during early development (Imamura et. al, 2010). An inconsistent regulation of cellular differentiation possibly caused by mutations in these genes could likely explain the wide range of phenotypes presented by the patient that involve abnormalities in skeletal, musculoskeletal, integumentary (as seen by low posterior hairline), and predicted connective tissue development. 

The apparent indirect interactions through *Myod1* between cell fate determining genes *Myocd* and *Chd2* in skeletal muscle phenotypes needs to be studied to confirm the true nature of the effects on these gene on the observed phenotypes. This could be approached by studying topologically associated domains (TADs) of these three genes using Hi-C data that would highlight any interactions between these genes. The analysis of the interacting domains in the patient to confirm whether these mutations affect the interactions of *Myocd* and *Chd2* with *Myod1* would strengthen this conclusion. In addition, understanding the structural changes of the proteins caused by these mutations would further build the mechanism by which these interactions have been affected and confirm genetic basis of the observed phenotypes seen in this patient. 

## Graphical
Figure 1: Decipher track of patient missense variant on gene *Chd2*. 
*Missense mutation is highlighted G>A. Missense constraint range shown to be less intolerant. The bottom of the image shows the highly conserved amino acid sequence of this gene across species.*
![](/project/project4_files/project4img1.jpg)

Figure 2: Decipher track of patient missense variant on gene *Myocd*. 
*Missense mutation is highlighted C>T. Missense constraint range shown to be more intolerant. The bottom of the image shows the conserved amino acid sequence of this gene across species.*
![](/project/project4_files/project4img2.jpg)

Figure 3: UCSC Genome Browser view of gene *Chd2*. 
*Genome Browser includes SNP data showing another missense mutation at the same position (15:93000537). The synonymous mutations are indicated by green SNPs and missense mutations indicated by yellow SNPs.*
![](/project/project4_files/project4img3.jpg)
 
Figure 4: UCSC Genome Browser view of gene *Myocd*. 
*Genome Browser includes SNP data showing another missense mutation around same position (17:12752657 - 17:12752658). The synonymous mutations are indicated by green SNPs and missense mutations indicated by yellow SNPs.*
![](/project/project4_files/project4img4.jpg)

Figure 5: gnomAD data of analysis of sequence variants in *Chd2* (left) and *Myocd* (right) gene from Decipher. 
*The data shows the occurrence of the specific alleles of these genes in different populations. No homozygotes appear to be observed in both gene studies. The sample size of these studies are indicated in the total allele number - 152096 for CHD2 and 152212 in *Myocd*.*
![](/project/project4_files/project4img5.png)
This project was completed under the guidance of Dr. Vokes during the Human Genetics (BIO 325T) course of spring 2021.

## References 

Alleman, B. W., Myking, S., Ryckman, K. K., Myhre, R., Feingold, E., Feenstra, B., Geller, F., Boyd, H. A., Shaffer, J. R., Zhang, Q., Begum, F., Crosslin, D., Doheny, K., Pugh, E., Pay, A. S. D., Østensen, I. H., Morken, N. H., Magnus, P., Marazita, M. L., . . . Murray, J. C. (2012). No observed association for mitochondrial SNPs with preterm delivery and related outcomes. Pediatric Research, 72(5), 539–544. https://doi.org/10.1038/pr.2012.112

Bezold, K. Y., Karjalainen, M. K., Hallman, M., Teramo, K., & Muglia, L. J. (2013). The genomics of preterm birth: from animal models to human studies. Genome Medicine, 5(4), 34. https://doi.org/10.1186/gm438

Capece, A., Vasieva, O., Meher, S., Alfirevic, Z., & Alfirevic, A. (2014). Pathway Analysis of Genetic Factors Associated with Spontaneous Preterm Birth and Pre-Labor Preterm Rupture of Membranes. PLoS ONE, 9(9), e108578. https://doi.org/10.1371/journal.pone.0108578

Carvill, G. L., Heavin, S. B., Yendle, S. C., McMahon, J. M., O’Roak, B. J., Cook, J., Khan, A., Dorschner, M. O., Weaver, M., Calvert, S., Malone, S., Wallace, G., Stanley, T., Bye, A. M. E., Bleasel, A., Howell, K. B., Kivity, S., Mackay, M. T., Rodriguez-Casero, V., . . . Mefford, H. C. (2013). Targeted resequencing in epileptic encephalopathies identifies de novo mutations in CHD2 and SYNGAP1. Nature Genetics, 45(7), 825–830. https://doi.org/10.1038/ng.2646

Di Renzo, G. C., Tosto, V., & Giardina, I. (2018). The biological basis and prevention of preterm birth. Best Practice & Research Clinical Obstetrics & Gynaecology, 52, 13–22. https://doi.org/10.1016/j.bpobgyn.2018.01.022

Harada, A., Okada, S., Konno, D., Odawara, J., Yoshimi, T., Yoshimura, S., Kumamaru, H., Saiwai, H., Tsubota, T., Kurumizaka, H., Akashi, K., Tachibana, T., Imbalzano, A. N., & Ohkawa, Y. (2012). Chd2 interacts with H3.3 to determine myogenic cell fate. The EMBO Journal, 31(13), 2994–3007. https://doi.org/10.1038/emboj.2012.136

Huang, J., Cheng, L., Li, J., Chen, M., Zhou, D., Lu, M. M., Proweller, A., Epstein, J. A., & Parmacek, M. S. (2008). Myocardin regulates expression of contractile genes in smooth muscle cells and is required for closure of the ductus arteriosus in mice. Journal of Clinical Investigation, 515–525. https://doi.org/10.1172/jci33304

Imamura, M., Long, X., Nanda, V., & Miano, J. M. (2010). Expression and functional activity of four myocardin isoforms. Gene, 464(1–2), 1–10. https://doi.org/10.1016/j.gene.2010.03.012

Khade, S. A., & Bava, A. K. (2018). Preterm premature rupture of membranes: maternal and perinatal outcome. International Journal of Reproduction, Contraception, Obstetrics and Gynecology, 7(11), 4499. https://doi.org/10.18203/2320-1770.ijrcog20184496

Kulkarni, S., Nagarajan, P., Wall, J., Donovan, D. J., Donell, R. L., Ligon, A. H., Venkatachalam, S., & Quade, B. J. (2008). Disruption of chromodomain helicase DNA binding protein 2 (CHD2) causes scoliosis. American Journal of Medical Genetics Part A, 146A(9), 1117–1127. https://doi.org/10.1002/ajmg.a.32178

Li, Y., Qiu, Q., Watson, S. S., Schweitzer, R., & Johnson, R. L. (2010). Uncoupling skeletal and connective tissue patterning: conditional deletion in cartilage progenitors reveals cell-autonomous requirements for Lmx1b in dorsal-ventral limb patterning. Development, 137(7), 1181–1188. https://doi.org/10.1242/dev.045237

Long, X., Creemers, E. E., Wang, D. Z., Olson, E. N., & Miano, J. M. (2007). Myocardin is a bifunctional switch for smooth versus skeletal muscle differentiation. Proceedings of the National Academy of Sciences, 104(42), 16570–16575. https://doi.org/10.1073/pnas.0708253104

Meester, J. A. N., Verstraeten, A., Schepers, D., Alaerts, M., Van Laer, L., & Loeys, B. L. (2017). Differences in manifestations of Marfan syndrome, Ehlers-Danlos syndrome, and Loeys-Dietz syndrome. Annals of Cardiothoracic Surgery, 6(6), 582–594. https://doi.org/10.21037/acs.2017.11.03

Romero, R., Friel, L. A., Velez Edwards, D. R., Kusanovic, J. P., Hassan, S. S., Mazaki-Tovi, S., Vaisbuch, E., Kim, C. J., Erez, O., Chaiworapongsa, T., Pearce, B. D., Bartlett, J., Salisbury, B. A., Anant, M. K., Vovis, G. F., Lee, M. S., Gomez, R., Behnke, E., Oyarzun, E., . . . Menon, R. (2010). A genetic association study of maternal and fetal candidate genes that predispose to preterm prelabor rupture of membranes (PROM). American Journal of Obstetrics and Gynecology, 203(4), 361.e1-361.e30. https://doi.org/10.1016/j.ajog.2010.05.026

Tchirikov, M., Schlabritz-Loutsevitch, N., Maher, J., Buchmann, J., Naberezhnev, Y., Winarno, A. S., & Seliger, G. (2017). Mid-trimester preterm premature rupture of membranes (PPROM): etiology, diagnosis, classification, international recommendations of treatment options and outcome. Journal of Perinatal Medicine, 46(5), 465–488. https://doi.org/10.1515/jpm-2017-0027
