---
title: "Reversing the effects of hypermethylation on BRCA1 by CRISPR/Cas9 technology."
author: "Ruth Raichur"
date: "2021-12-15T22:55:05-04:00"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, eval = TRUE, fig.align = "center", warning = F, message = F,
tidy=TRUE, tidy.opts=list(width.cutoff=60), R.options=list(max.print=100))
```

  Breast cancer susceptibility gene 1 (*BRCA1*) is a tumor suppressor gene involved in the pathogenesis of tumors specific to breast and ovarian cancer. The normal biological functions of this gene are primarily associated with controlling cell proliferation by DNA damage repair and cell cycle checkpoint control that include promoting apoptosis and controlling hypermethylation (Krasteva et al., 2012). Mechanisms that cause loss of function in *BRCA1* include somatic mutations, DNA hypermethylation, dysregulation of transcriptional activators, and *BRCA1* locus binding. Historically, links between loss of function in *BRCA1* by hypermethylation and sporadic breast/ovarian cancer patients have been elucidated by meta-analyses studies (Krasteva et al., 2012). Methylation is a heritable mechanism that controls gene expression by marking dinucleotide repeats CG in genes, known as CpG islands, with a methyl group which causes repression. In hypermethylated cells, CpG islands around the promoter regions of genes are enriched with methyl groups in amounts more significant than normal, causing transcriptional repression that inhibits transcription factor binding. In cases of sporadic breast cancer, studies found the loss of function in *BRCA1* was primarily caused by promoter hypermethylation resulting in gene silencing and tumor initiation (Krasteva et al., 2012). Thus, an approach to preventing tumor progression could be by removing promoter hypermethylation. The clustered regulatory interspaced short palindromic repeat (CRISPR)-Cas9 system is an editing tool used for precise gene editing. The homology-directed repair (HDR) system modifies genes using a repair template of choice after Cas9 induces a targeted break at the desired site directed by guide RNA. It has been observed that the methylation status of genes does not interfere with this system and therefore posits the system’s ability to induce epigenetic modification (Sung & Yim, 2020). Understanding the effects of promoter hypermethylation of *BRCA1* in sporadic breast/ovarian cancer could assist in using the CRISPR-Cas9 system as potential gene therapy with careful considerations.
  
  While the causation of breast/ovarian cancer can be hereditarily linked to mutations in regulatory genes, a strong correlation between hypermethylation of the *BRCA1* gene has been identified in sporadic cases of breast/ovarian cancer, cases with no family history of the disease (Krasteva et al., 2012). This finding suggests that many sporadic cases are not due to random genetic mutations but the malfunctioning of gene regulatory mechanisms. A cohort study was performed in a group of 135 Bulgarian patients to determine a significant correlation between the level of methylation in the promoter region of *BRCA1* and sporadic breast/ovarian cancer. Methylation-specific PCR identified the presence of methylation in the isolated tumor samples, and multiple statistical methods were used to analyze the clinical significance (Krasteva et al., 2012). They found that hypermethylation of the *BRCA1* promoter was present in 17.04% of the Bulgarian patients. The implications of this study provide more data towards understanding *BRCA1* tumorigenesis in breast/ovarian cancer, especially specific to the Bulgarian population. The results ultimately define a link between hypermethylation of *BRCA1* and the occurrence of breast/ovarian cancer.
  
  Discussions of similar studies show different strengths of associations between the hypermethylation of *BRCA1* in sporadic breast/ovarian cancer, establishing a strong correlation primarily in sporadic cases. A meta-analysis study evaluated the link between hypermethylation of the promoter of *BRCA1* and its gene expression to understand the etiology of breast cancer in sporadic cases. *BRCA1* methylation and mRNA expression was detected by bisulfite sequencing and quantitative polymerase chain reaction (qPCR), respectively, in 49 surgically resected cancerous and paired non-cancerous samples from patients with breast cancer. The study concluded that *BRCA1* silencing by hypermethylation is a likely cause of sporadic breast cancer cases. *BRCA1* hypermethylation was identified in 49% of the sporadic breast/ovarian cancer cases, with a significantly higher occurrence in breast cancer tissues (Li et al., 2015). A strong negative correlation between methylation status and *BRCA1* mRNA expression in sporadic cases was identified, suggesting hypermethylation was related to lower mRNA expression (Li et al., 2015). This study demonstrates a strong link between promoter hypermethylation of *BRCA1* and tumorigenesis due to reduced expression of the tumor suppressor in sporadic breast cancer cases. This cause-and-effect link suggests that demethylating the *BRCA1* promoter could potentially reactivate tumor suppressor expression and prevent tumor progression. A CRISPR Cas9 system that removes the hypermethylation on the *BRCA1* promoter poses as a potential therapeutic for sporadic breast/ovarian cancer with cases of high promoter methylation.
  
  The link between promoter hypermethylation and the emergence of tumors has already influenced existing cancer treatments. They include chemotherapies and anticancer drugs that cause epigenetic modifications in some cancers, such as 5-fluorouracil (5-FU) known to change epigenetic modifications in gastric cancer patients with general methyltransferase inhibitors and demethylases (Sung & Yim, 2020). In a review, Sung & Yim (2020) evaluate the potential of CRISPR-mediated promoter de/methylation technologies for gene therapies by controlling regulation. The authors present CRISPR systems for gene therapies involved with reversing methylation control, similar to the argument discussed in this paper. These ideas are explored in a much broader context, studying the epigenetics of tumor progression, and discussing how an early event in carcinogenesis is DNA methylation dysregulation. They introduce the CRISPR system as a novel method to modify the methylation in these patients, suggesting modifications to the system that could boost methylation efficiency and accuracy. Some of these methods discussed include adapting a bacterial (Mollicutes spiroplasma) DNA methyltransferase (MQ1) for site-directed promoter methylation, constructing a vector expressing dCas9 fused to a single-chain DNA methyltransferase DNMT3A-3 L or combining it with a histone methyltransferase Ezh2 or KRAB to increase the methylation activities (Sung & Yim, 2020). Finally, the review discusses the importance of combining histone and DNA methyltransferases with the CRISPR-Cas9 system to ensure optimal epigenetic modifications (Sung & Yim, 2020). The paper concludes that this novel mechanism can be a valuable tool in cancer therapies as it is specific and irreversible, highlighting that these changes will not succumb to reversal by DNA methyltransferase. This modified CRISPR system combined with histone and DNA methyltransferases could be utilized to remove the hypermethylation on the *BRCA1* promoter. Based on the previously discussed epigenetic influences on tumor progression in breast/ovarian cancer, the effect would likely decrease tumor progression, making this system a viable option for treatment.
  
  While exploring the use of the CRISPR-Cas9 system to modulate promoter methylation, many studies have been performed to develop the most efficient CRISPR model to achieve this task. One such approach was performed by Xu et al. (2016) using the CRISPR model to upregulate selected transcription target genes by demethylating the CpG islands on the promoter regions of these genes. To do this, a dCas9-mediated demethylation system which consisted of a dCas9 fused to a Tet1 catalytic domain (Tet1-CD), was constructed (Xu et al., 2016). They then modified the single guide RNA (sgRNA) into a conventional guide RNA (sgRNA 2.0) that would facilitate the tethering of this dCas9-mediated demethylation system to the specific site. Each sgRNA was modified to target promoter CpG regions of the gene known to be silenced by hypermethylation, RANKL. The upregulating impact of each of the different sgRNAs was determined by measuring the mRNA transcribed by RANKL after transfection of HEK-293FT cells with the CRISPR system. By comparing the dCas9-Tet1-CD/sgRNA system to a catalytically dead form of Tet1-CD in the dCas9, the scientists confirmed that the upregulation observed resulted from using the dCas9 system. They found that the sgRNA 2.0 and the dCas9-Tet1-CD/sgRNA system significantly induced gene activation of RANKL, with an approximate 1.5-fold increased expression (Xu et al., 2016).
  
  After establishing the sgRNA/dCas9-Tet1-CD system, other researchers wished to evaluate the effectiveness of this system on site-specific demethylation. They specifically targeted the promoter of Oct4, a Yamanaka factor gene highly expressed in stem cells that is generally silenced by hypermethylation in differentiated cells. Using a donor plasmid that carried non-methylated dinucleotide sequences that would replace the CpG sites, they generated a CRISPR/Cas9 mediated knock-in system that targeted the promoter region of Oct4 in differentiated cells. The effectiveness of this system was then evaluated by measuring the frequency of methylation remaining at each CpG site in this region by bisulfite sequencing. The system was also used with histone methyltransferase inhibitors and histone deacetylases in NIH3T3 cells to determine the most effective method of gene reactivation. They found that the sgRNA/dCas9-Tet1 system only worked effectively in demethylating the promoter when used in conjunction with histone methyltransferase inhibitors and histone deacetylases in NIH3T3 cells, suggesting the need to suppress histone modifications when using this system. Additionally, specific allele substitutions using the CRISPR system, such as an ‘AG’ substituted for a CpG site, were observed to have more pronounced effects on gene activation (Kang et al., 2019). These specific substitutions led to approximately a 1.5 to 3-fold increase in Oct4 gene transcription in cell clones with these dinucleotide substitutions in the upper promoter region. The researchers concluded that this CRISPR system would be an effective DNA demethylation strategy in conjunction with histone methyltransferase inhibitors and histone deacetylases. Ultimately, their research confirmed the effectiveness of inducing site-specific mutations in one or adjacent sites. It would be most helpful in ex vivo gene therapy (Kang et al., 2019), that could be used in reactivating the promoter region of *BRCA1*, bearing in mind the specific modifications and additional requirements.
  
  The CRISPR/Cas9 system has proven to be an influential gene-editing tool that now has capabilities in mediating epigenetic modifications under the right conditions. This system could potentially be highly beneficial as a treatment for breast/ovarian cancers under specific criteria. While there are many causes for breast/ovarian cancer, the most common sporadic ovarian/breast cancer occurrences are linked to promoter hypermethylation of the tumor suppressor gene, *BRCA1* (Krasteva et al., 2012). Studies have confirmed a statistically significant correlation between *BRCA1* expression and promoter hypermethylation, showing that increased hypermethylation results in low levels of *BRCA1* expression giving rise to tumor development (Li et al., 2015). Experiments involving the use of a CRISPR/Cas9 system to epigenetically reactive genes have been proven to be effective, efficient, and specific (Xu et al., 2016). However, studies have shown that certain modifications, such as fusing Cas9 to a Tet1 catalytic domain and enhancing the sgRNA, are essential to reactivate genes significantly (Xu et al., 2016). Additionally, to account for histone modifications regulating promoter regions, this therapy will likely be most effective when used with histone methyltransferase inhibitors and histone deacetylases (Kang et al., 2019). With these modifications, experiments have shown to effectively reactivate genes that have previously been silenced by promoter hypermethylation, similar to the case of *BRCA1* in sporadic breast/ovarian cancers. Using the sgRNA/dCas9-Tet1-CD CRISPR system to increase *BRCA1* expression poses itself as a potential therapy for sporadic breast/ovarian cancer patients where hypermethylation on the promoter has been previously established. Further research will be required to refine the technology to increase efficiency in this context and establish an effective delivery method through stem cell treatment or other known methods. Nevertheless, implications of this technology can be used to revolutionize cancer therapy for not only sporadic breast/ovarian cancer but also other diseases caused by promoter methylation silencing. 

This paper was completed under the guidance of Dr. Bruner during the Lab Experiences in Genetics (BIO 325L) course of fall 2021.

## References
Kang, J. G., Park, J. S., Ko, J.-H., & Kim, Y.-S. (2019). Regulation of gene expression by altered promoter methylation using a CRISPR/Cas9-mediated epigenetic editing system. *Scientific Reports, 9*(1), 11960. https://doi.org/10.1038/s41598-019-48130-3

Krasteva, M. E., Bozhanov, S. S., Antov, G. G., Gospodinova, Z. I., & Angelova, S. G. (2012). Breast cancer patients with hypermethylation in the promoter of *BRCA1* gene exhibit favorable clinical status. *Neoplasma, 59*(01), 85–91. https://doi.org/10.4149/neo_2012_011

Li, Q., Wei, W., Jiang, Y., Yang, H., & Liu, J. (2015). Promoter methylation and expression changes of *BRCA1* in cancerous tissues of patients with sporadic breast cancer. *Oncology Letters, 9*(4), 1807–1813. https://doi.org/10.3892/ol.2015.2908

Sung, C. K., & Yim, H. (2020). CRISPR-mediated promoter de/methylation technologies for gene regulation. *Archives of Pharmacal Research, 43*(7), 705–713. https://doi.org/10.1007/s12272-020-01257-8

Xu, X., Tao, Y., Gao, X., Zhang, L., Li, X., Zou, W., Ruan, K., Wang, F., Xu, G., & Hu, R. (2016). A CRISPR-based approach for targeted DNA demethylation. *Cell Discovery, 2*(1), 16009. https://doi.org/10.1038/celldisc.2016.9
