<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="author" content="Ruth Raichur" />
    
    <link rel="shortcut icon" type="image/x-icon" href="/img/favicon.ico">
    <title>Proposed Gli Enhancer Target Gene Osr2 Involved In Embryonic Limb Morphogenesis By Hedgehog Signal Regulation.</title>
    <meta name="generator" content="Hugo 0.92.1" />
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="/css/main.css" />
    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,400,200bold,400old" />
    
    <!--[if lt IE 9]>
			<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
			<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
		<![endif]-->

    
  </head>

  <body>
    <div id="wrap">
      
      <nav class="navbar navbar-default">
  <div class="container">
    <div class="navbar-header">
      <a class="navbar-brand" href="/"><i class="fa fa-home"></i></a>
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    <div class="navbar-collapse collapse" id="navbar">
      <ul class="nav navbar-nav navbar-right">
      
        
        <li><a href="/projects/">PROJECTS</a></li>
        
        <li><a href="/resume/">RESUME</a></li>
        
      
      </ul>
    </div>
  </div>
</nav>

      <div class="container">
        <div class="blog-post">
          <h3>
            <strong><a href="/project/project3/">Proposed Gli Enhancer Target Gene Osr2 Involved In Embryonic Limb Morphogenesis By Hedgehog Signal Regulation.</a></strong>
          </h3>
        </div>
 
<div class="blog-title">
          <h4>
         March 12, 2021 
            &nbsp;&nbsp;
            
          </h4>
        </div>

        <div class="panel panel-default">
          <div class="panel-body">
            <div class="blogpost">
              


<P style="page-break-before: always">
<div style="page-break-after: always;"></div>
<div id="summary" class="section level2">
<h2>Summary</h2>
<p>The purpose of this study was to identify a gene with intersecting Gli binding sites and H3K27ac hallmarks that have associations with human limb abnormalities. The intersection between HH responsive H3K27ac sites and Gli binding region identified 340 active Gli enhancers target genes with about 20+ sites mostly involved with limb, digit, and embryonic morphogenesis. One of these genes was selected for an in-depth analysis to determine the likelihood of Gli enhancers mediated Hedgehog signaling in this gene that regulates limb development. Using the intersection on GREAT software, <em>Osr2</em> was identified to have an effect on human embryonic limb morphogenesis and was chosen as the gene of interest. Upon further study, there appeared to be strong evidence suggesting that <em>Osr2</em> is a potential Gli target gene and is hypothesized to be regulated by the Hedgehog mechanism.</p>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>The Sonic Hedgehog (Shh) gene encodes for a protein that has the properties of a morphogen specifying the positional values across the antero-posterior axis of the limb (Tickel and Baker, 2012). On a transcriptional level, research has shown this signaling controls cellular differentiation in both dose dependence and time of Shh exposure. It has been studied to have long range effects as well as work in collaboration with numerous other signaling pathways (Dessaud et.al, 2007). Defects in the response to altered Shh signaling have been found to underlie inherited disorders that include polydactyly and several other clinical conditions with congenital limb defects (Tickel and Towers, 2017).</p>
<p>Enhancers are cis-regulatory elements that coordinate transcription in the cell (Andersson and Sandelin, 2019). General epigenetic hallmarks of enhancers are determined by various histone modifications such as methylation and acetylation markers. The acetylation of histone H3 (H3K27ac) is a mark for active enhancers while trimethylation of this histone (H3K27me3) is a mark for both active and poised enhancers (Yao et.al, 2020). Interestingly, Gli proteins have been seen to regulate gene expression by managing enhancer activity, controlling H3K27ac enrichment (Lex et.al, 2020). This suggests that a reasonable method to identify their target regions that likely have a relationship with Shh signaling would be to identify HH responsive active enhancers (enriched with H3K27ac) using ChiP-seq analysis and intersect it with identified Gli binding regions.</p>
<p>This intersection between Gli binding regions and HH responsive H3K27ac sites on a mouse dataset revealed a potential target gene, <em>Osr2</em>. This gene is known to be a member of highly conserved zinc-finger transcription factors (Orgeur et.al, 2018) that most likely contributes to normal limb development in humans as they have been observed to be required factors in leg joint formation in Drosophila and expressed in synovial joint cells in mice (Gao et.al, 2011).</p>
</div>
<div id="description-of-findings" class="section level2">
<h2>Description of findings</h2>
<p>ChiP-Seq datasets (Lex et al, 2020) were used on the UCSC Genome Browser for this study. The first dataset contained all 2,113 HH-responsive H3K27ac enriched regions in the mouse limb bud. This was intersected with a dataset that mapped all the Gli Binding regions (GBR) that have active enhancer markers. The table browser tool on the UCSC Genome Browser was used to perform an intersection that mapped any overlap between these two tracks (Session links included in the ‘Graphical’ section). GREAT was used to analyze the intersection, predicting the target genes associated with the near active enhancers. These regions are identified as potential Gli binding regions. This software provided a summary table of all genomic region-gene association table (Table 2) which identified regions associations and their chromosome coordinates that would help further verify their location relative to the Gli binding regions.</p>
<p>The Gli3-H3K27ac intersection yielded a total of 280 intersecting regions (Table 1) with 340 genes (Table 2). This intersection analyzed on GREAT revealed further information about the target genes sorted by phenotype such as GO Biological processes, GO Molecular Functions and GO Cellular Component. In terms of GO Biological process, the intersection revealed about 20+ sites mostly involved with limb, digit, and embryonic morphogenesis. There were 8 terms for GO Molecular Function and no terms for GO Cellular Component. 20+ terms were found in the human phenotype. Additionally, around 80% of genomic regions had 2 associated genes within them and the size range of enhancers was found to be large, almost +/- 500,000bps. However, a majority of the GBRs were present between 50 and 500 kbps away from the TSS of the identified genes. (Figure 1).</p>
<p>Using the list of GO biological processes that identified intersections between region-gene associations that had involvement in specific biological phenotypes, <em>Osr2</em> was identified under the “embryonic limb morphogenesis” phenotype. Its location was further verified using Genome browser by comparing the location of <em>Osr2</em> to the two tracks and the intersection. The <em>Osr2</em> gene promoter was observed to overlap a Gli binding region and an HH responsive H3K27ac site (Figure 2) suggesting its likelihood of being an active Gli target gene. Since <em>Osr2</em> met the two criteria of firstly, having a phenotype associated with human limb abnormalities or functions and then overlapping a Gli binding site with HH responsives site, the <em>Osr2</em> was picked as the gene of interest.</p>
<p><em>Osr2</em> is a member of the odd-skipped gene family seen to act as a regulator of palatal growth and patterning of the endoderm, heart, teeth, bones and synovial joints in the limbs in mammals (Neto et.al, 2012). There are no known human limb defects caused by mutations in this gene but <em>Osr2</em> is seen to be expressed with Osr1 during later stages of limb development in developing joints suggesting a crucial role in human limb development (Neto et.al, 2012). <em>Osr2</em>’s role in regulation of limb development is seen to be evolutionarily conserved as reduction in <em>Osr2</em> function in zebrafish was seen to impair fin development by failure of gene maintenance in the developing pectoral fin bud (Neto et al, 2012).</p>
</div>
<div id="discussion-and-conclusion" class="section level2">
<h2>Discussion and Conclusion</h2>
<p>With the understanding of the relationship between Gli proteins in their regulation of H3K27ac enrichment, a prediction of potential Gli target sites regulated by HH signaling in the mouse genome could be made by running an intersection that mapped HH responsive H3K27ac sites against all Gli binding regions (Lex et.al, 2020). The strategy analyzed the Gli3-H3K27ac intersection on GREAT and identified 340 potential Gli target genes (Table 2). This is perhaps an overestimation considering the strategy only consisted of the single criteria of an HH responsive H3K27ac sites having any overlap with a Gli binding region. Therefore, it is unlikely that all identified intersecting genes are Gli target regions.</p>
<p>This overestimation is a limitation given that Gli target genes can be determined by a plethora of factors some of which are still unknown. In addition, the distance from the enhancers must be taken into consideration for all identified genes. Only around 60% of genes identified had a Gli binding region located between 50 and 500 kb away from their TSS (Figure 1c). This suggests that at least the 10% that is located at a distance greater than 500 kb away from the Gli binding region have a low probability of being a Gli target gene (Figure 1c). However, this does not entirely rule out the possibility since Gli is known to be a variable range enhancer. Ultimately, given the percentage of genes at larger distances from the GBR and the variable factors that have not been taken into account, it is safe to predict GREAT over-reported the number of regions.</p>
<p>The selected target gene, <em>Osr2</em>, is 102 b away from the Gli Binding region which is within the common range suggesting a high likelihood of it being a Gli target gene. The genome browser view of <em>Osr2</em> identifies two Gli3 binding region located around the promoter region of the gene. One is presumed to be active since it overlaps with an HH responsive H3K27ac region while the other is assumed to be inactive since does not. In addition, there is one H3K27ac region (active enhancer region) that is not bound by Gli3 present within about 100kb upstream from the end of <em>Osr2</em> (Figure 2). Since the active enhancer is upstream of the gene it is unlikely that this additional unbound enhancer plays a significant role in the expression of <em>Osr2</em>.</p>
<p>While it is known that <em>Osr2</em> plays a crucial role in limb development, so far, the only known human abnormalities where <em>Osr2</em> is identified as the main cause is associated with the cleft palate (Zhou et al, 2013). However, null mutants for <em>Osr2</em> were shown to have a subtle effect in synovial joint development and cause fusion of multiple joints in mice (Gao et al, 2011). There is strong evidence that this occurs in an HH dependent fashion since <em>Osr2</em> in developing palatal mesenchyme as there was an observed down regulation in Smo mutants suggesting that <em>Osr2</em> is downstream in the Shh pathway (Lan and Jiang, 2009). Given the HH dependence in palatal development, the HH dependence of <em>Osr2</em> regulation in limb development could be a potential area of study as there are strong grounds for a relationship.</p>
<p>To experimentally confirm this role of Shh signaling, In-situ hybridization using cDNA complementary to the <em>Osr2</em> gene can be performed to study it’s localization in the antero-posterior axis of the limb in an Shh knockout embryo compared to WT. We would expect to see decreased or no localization of <em>Osr2</em> in the Shh knockout compared to WT. This would suggest HH dependence of <em>Osr2</em>’s role in limb morphogenesis.</p>
</div>
<div id="graphical" class="section level2">
<h2>Graphical</h2>
<p>Table 1: Custom tracks from UCSC Genome Browser used for analysis.
<em>“User Track” is the intersection track created between the Gli3 sites and HH responsive H3K27ac sites that show 280 sites. “Gli_Limb_Binding_Endogenous_mm10” is track mapping the 7282 Gli3 binding regions on the mouse genome while the “HH_responsive_H3K27ac_mm10” shows the Chip-Seq data mapping the 2113 hedgehog responsive H3K27 acetylated binding regions on the same genome both obtained from Lex et al, 2020.</em>
<img src="/project/project3_files/project3img1.jpg" /></p>
<p>Figure 1: Graphs a, b and c describing locations of GBRs from TSS of target genes identified. <em>Graph a shows number of associated genes per region, 80% of regions have 2 associated genes. Graph b shows the region-gene associations based on orientation and distance of the GBRs to the transcription start site (TSS) of each target gene, mostly located +/- 50 to 500kb. Graph c shows describes the same data as graph b but in absolute distance to the TSS, majority between 50 to 500 kb.</em>
<img src="/project/project3_files/project3img2.jpg" /></p>
<p>Figure 2: Confirming <em>Osr2</em> likelihood of being a Gli3 target gene.
<em>Using UCSC Genome Browser to visualize positions of Osr2 relative to Gli limb binding regions and HH responsive regions. Track colored in red indicates Gli3 limb binding region, track colored in green is hedgehog responsive region. The two black track indicates location of Osr2 gene with promoter region and without.</em>
<img src="/project/project3_files/project3img3.jpg" /></p>
<p>UCSC Genome Browser mapping two datasets from Lex et al, 2020 <a href="https://genome.ucsc.edu/s/ruthraichur/Gli3%20limb%20binding%20endogenous_mm10"> Session Link </a></p>
<p>UCSC Genome Browser mapping the intersection: <a href = http://genome.ucsc.edu/s/ruthraichur/Gli3_HH_intersection> Session Link </a></p>
<p>This project was completed under the guidance of Dr. Vokes during the Human Genetics (BIO 325T) course of spring 2021.</p>
</div>
<div id="references" class="section level2">
<h2>References</h2>
<p>Andersson, R., &amp; Sandelin, A. (2019). Determinants of enhancer and promoter activities of regulatory elements. Nature Reviews Genetics, 21(2), 71–87. <a href="https://doi.org/10.1038/s41576-019-0173-8" class="uri">https://doi.org/10.1038/s41576-019-0173-8</a></p>
<p>Dessaud, E., Yang, L. L., Hill, K., Cox, B., Ulloa, F., Ribeiro, A., Mynett, A., Novitch, B. G., &amp; Briscoe, J. (2007). Interpretation of the sonic hedgehog morphogen gradient by a temporal adaptation mechanism. Nature, 450(7170), 717–720. <a href="https://doi.org/10.1038/nature06347" class="uri">https://doi.org/10.1038/nature06347</a></p>
<p>Gao, Y., Lan, Y., Liu, H., &amp; Jiang, R. (2011). The zinc finger transcription factors Osr1 and Osr2 control synovial joint formation. Developmental Biology, 352(1), 83–91. <a href="https://doi.org/10.1016/j.ydbio.2011.01.018" class="uri">https://doi.org/10.1016/j.ydbio.2011.01.018</a></p>
<p>Lan, Y. (2004). Odd-skipped related 2 (Osr2) encodes a key intrinsic regulator of secondary palate growth and morphogenesis. Development, 131(13), 3207–3216. <a href="https://doi.org/10.1242/dev.01175" class="uri">https://doi.org/10.1242/dev.01175</a></p>
<p>Lan, Y., &amp; Jiang, R. (2009). Sonic hedgehog signaling regulates reciprocal epithelial-mesenchymal interactions controlling palatal outgrowth. Development, 136(8), 1387–1396. <a href="https://doi.org/10.1242/dev.028167" class="uri">https://doi.org/10.1242/dev.028167</a></p>
<p>Lex, R. K., Ji, Z., Falkenstein, K. N., Zhou, W., Henry, J. L., Ji, H., &amp; Vokes, S. A. (2020). GLI transcriptional repression regulates tissue-specific enhancer activity in response to Hedgehog signaling. ELife, 9, 1. <a href="https://doi.org/10.7554/elife.50670" class="uri">https://doi.org/10.7554/elife.50670</a></p>
<p>Navet, S., Andouche, A., Baratte, S., &amp; Bonnaud, L. (2009). Shh and Pax6 have unconventional expression patterns in embryonic morphogenesis in Sepia officinalis (Cephalopoda). Gene Expression Patterns, 9(7), 461–467. <a href="https://doi.org/10.1016/j.gep.2009.08.001" class="uri">https://doi.org/10.1016/j.gep.2009.08.001</a></p>
<p>Neto, A., Mercader, N., &amp; Gomez-Skarmeta, J. L. (2011). The osr1 and osr2 genes act in the pronephric anlage downstream of retinoic acid signaling and upstream of wnt2b to maintain pectoral fin development. Development, 139(2), 301–311. <a href="https://doi.org/10.1242/dev.074856" class="uri">https://doi.org/10.1242/dev.074856</a></p>
<p>Orgeur, M., Martens, M., Leonte, G., Nassari, S., Bonnin, M.-A., Börno, S. T., Timmermann, B., Hecht, J., Duprez, D., &amp; Stricker, S. (2018). Genome-wide strategies identify downstream target genes of chick connective tissue-associated transcription factors. Development, 145(7), dev161208. <a href="https://doi.org/10.1242/dev.161208" class="uri">https://doi.org/10.1242/dev.161208</a></p>
<p>Tickle, C., &amp; Barker, H. (2012). The Sonic hedgehog gradient in the developing limb. Wiley Interdisciplinary Reviews: Developmental Biology, 2(2), 275–290. <a href="https://doi.org/10.1002/wdev.70" class="uri">https://doi.org/10.1002/wdev.70</a></p>
<p>Tickle, C., &amp; Towers, M. (2017). Sonic Hedgehog Signaling in Limb Development. Frontiers in Cell and Developmental Biology, 5, 1. <a href="https://doi.org/10.3389/fcell.2017.00014" class="uri">https://doi.org/10.3389/fcell.2017.00014</a></p>
<p>Yao, J., Chen, J., Li, L.-Y., &amp; Wu, M. (2020). Epigenetic plasticity of enhancers in cancer. Transcription, 11(1), 26–36. <a href="https://doi.org/10.1080/21541264.2020.1713682" class="uri">https://doi.org/10.1080/21541264.2020.1713682</a></p>
<p>Zhou, J., Gao, Y., Lan, Y., Jia, S., &amp; Jiang, R. (2013). Pax9 regulates a molecular network involving Bmp4, Fgf10, Shh signaling and the Osr2 transcription factor to control palate morphogenesis. Development, 140(23), 4709–4718. <a href="https://doi.org/10.1242/dev.099028" class="uri">https://doi.org/10.1242/dev.099028</a></p>
</div>
<div id="supplemental-data" class="section level2">
<h2>Supplemental Data</h2>
<p>Table 2: Potential Nearby Gli3 Target Genes and their chromosomal coordinates.</p>
<p><em>All genomic region-gene association table for the Gli3-H3K27ac intersection. GREAT was unable to provide chromosomal positions (“unnamed”) however, coordinates to the TSS were provided.</em></p>
<pre class="r"><code>library(readr)
genomic_regions &lt;- read_csv(&quot;/Users/ruthraichur/Desktop/website/content/project/project3.csv&quot;)
data.frame(genomic_regions)</code></pre>
<pre><code>##             Gene                                                Position
## 1  1600014C23Rik   unnamed (-203356), unnamed (-23606), unnamed (-22681)
## 2  1700018B08Rik                    unnamed (+239979), unnamed (+241204)
## 3  1700129C05Rik                    unnamed (+154568), unnamed (+155368)
## 4  2810428I15Rik                                        unnamed (+11777)
## 5  4930546C10Rik                                       unnamed (+322366)
## 6  5730508B09Rik                                       unnamed (-285501)
## 7  6030458C11Rik                    unnamed (+143507), unnamed (+193407)
## 8  A330070K13Rik                                       unnamed (-553494)
## 9        A3galt2                                        unnamed (+45192)
## 10      AI606181                                        unnamed (+19812)
## 11        Actbl2                                        unnamed (-84638)
## 12        Adgrb1                                       unnamed (-359320)
## 13          Aff3                    unnamed (-131740), unnamed (-129990)
## 14         Agap1                                       unnamed (+394319)
## 15         Agtr2                                        unnamed (-50269)
## 16         Alg13                                          unnamed (+821)
## 17       Angptl8                                        unnamed (+11740)
## 18       Ankrd16                                       unnamed (+404084)
## 19       Ankrd44                                         unnamed (-4988)
## 20          Aqp4                                       unnamed (+247434)
## 21      Arhgap15                                       unnamed (+592955)
## 22      Arhgap24                    unnamed (+158809), unnamed (+159409)
## 23       Arhgap9                      unnamed (+15993), unnamed (+17093)
## 24          Arsb                    unnamed (+194445), unnamed (+195370)
## 25         Asb13                                        unnamed (+79093)
## 26          Ass1                    unnamed (-130482), unnamed (-129682)
## 27         Astn1                                       unnamed (+498527)
## 28        Atp2b4                                        unnamed (+58491)
## 29        Atp5g3    unnamed (-67525), unnamed (-65675), unnamed (-41775)
## 30       Atp6ap2                    unnamed (-368651), unnamed (-367776)
## 31      Atp6v0d1                                        unnamed (+45122)
## 32         Auts2                    unnamed (-591501), unnamed (-590376)
## 33      BC034090                                        unnamed (+19890)
## 34         Bcat1 unnamed (+259937), unnamed (+260837), unnamed (+261287)
## 35          Bcor                    unnamed (-139472), unnamed (-138597)
## 36          Bdnf                                         unnamed (+2975)
## 37       Bhlhe40                                       unnamed (-203929)
## 38          Bmp7                                       unnamed (+265321)
## 39      Calcoco1    unnamed (-30772), unnamed (-29772), unnamed (-25022)
## 40        Calml3                                        unnamed (+91191)
## 41        Calml4                         unnamed (-404), unnamed (+1171)
## 42         Casp8                                        unnamed (+30648)
## 43       Cbfa2t3                                        unnamed (+50359)
## 44         Cbln4                                       unnamed (+308641)
## 45       Ccdc182 unnamed (+395592), unnamed (+396742), unnamed (+400192)
## 46         Ccnd1                         unnamed (-1006), unnamed (+744)
## 47          Cct7                                        unnamed (+40711)
## 48          Cd83                                       unnamed (+107500)
## 49         Cdh11                      unnamed (+95892), unnamed (+96717)
## 50         Cdh26                                        unnamed (+83543)
##  [ reached &#39;max&#39; / getOption(&quot;max.print&quot;) -- omitted 290 rows ]</code></pre>
</div>

            
        <hr>         <div class="related-posts">
                <h5>Related Posts</h5>
                
              </div> 
            </div>
          </div>

   <hr>  <div class="disqus">
  <div id="disqus_thread"></div>
  <script type="text/javascript">

    (function() {
      
      
      if (window.location.hostname == "localhost")
        return;

      var disqus_shortname = '';
      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
      dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  <a href="http://disqus.com/" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
</div> 
        </div>
      </div>
    </div>

    
    <footer>
  <div id="footer">
    <div class="container">
      <p class="text-muted">&copy; All rights reserved. Powered by <a href="https://gohugo.io/">Hugo</a> and
      <a href="http://www.github.com/nurlansu/hugo-sustain/">sustain</a> with ♥</p>
    </div>
  </div>
</footer>
<div class="footer"></div>


<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>

<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
<script src="/js/docs.min.js"></script>
<script src="/js/main.js"></script>

<script src="/js/ie10-viewport-bug-workaround.js"></script>


    
  </body>
</html>
